miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppConfig.kt

```diff
@@ -0,0 +1,31 @@
+package com.rakuten.tech.mobile.miniapp.view
+
+import com.rakuten.tech.mobile.miniapp.MiniAppSdkConfig
+import com.rakuten.tech.mobile.miniapp.file.MiniAppFileChooser
+import com.rakuten.tech.mobile.miniapp.js.MiniAppMessageBridge
+import com.rakuten.tech.mobile.miniapp.navigator.MiniAppNavigator
+import com.rakuten.tech.mobile.miniapp.sdkExceptionForInvalidArguments
+
+/**
+ * This represents the configuration settings for the Mini App.
+ * @property miniAppSdkConfig configuration for Mini App SDK.
+ * @param miniAppMessageBridge the interface for communicating between host app & mini app.
+ * @param miniAppNavigator allow host app to handle specific urls such as external link.
+ * @param miniAppFileChooser allow host app to get the file path while choosing file inside the webview.
+ * @param queryParams the parameters will be appended with the miniapp url scheme.
+ */
+data class MiniAppConfig(
```

Comment:
(Optional) I would suggest to rename this class to `MiniAppViewConfig`.

Reply:
for sdk configuration we use `MiniAppSdkConfig` and for miniapp configuration we use `MiniAppConfig`, it feels more appropriate i think. What do you think? @khairul-alam-licon @laubengaier

Reply:
Think both are descriptive and serve the purpose, I went with `MiniAppConfig` as well for now but alam has a point because `MiniApp` will continue to exist so MiniApp -> MiniAppSdkConfig and MiniAppView -> MiniAppConfig could be confusing